doc/aspice_swe3_4/example/iceoryx_component/source/example_component/example_base_class.inl

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "example_base_class.hpp"
```

Comment:
Wasn't the consensus that the header should not be included? Just asking, I don't have strong feeling either way.

Reply:
I think it should not be included here.
If you do the hpp includes the inl, which then includes the hpp, which includes the inl and so on. Just protected by their guards.
It's unnecessary, kind of confusing, so don't include it here.